src/nrfx/hal/nrf_clock.h

```diff
@@ -42,8 +42,7 @@
 #ifndef BS_NRF_CLOCK_H__
 #define BS_NRF_CLOCK_H__
 
-#include "nrf_soc_if.h"
-#include "../drivers/nrfx_common.h"
+#include <nrfx.h>
```

Comment:
Dont forget about this one :)

Reply:
I changed the approach and reverted the changes that introduced the need for this file. And the PR became even simpler.